docs/src/components/DocsPageBuilder/BpkMarkdownRenderer.js

```diff
@@ -147,7 +147,7 @@ const BpkMarkdownRenderer = (props: Props) => {
 
 BpkMarkdownRenderer.propTypes = {
   source: PropTypes.string.isRequired,
-  darkBackground: PropTypes.boolean,
+  darkBackground: PropTypes.bool,
```

Comment:
Fixing this as was throwing an error in the browser as being the incorrect prop type